CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.11.0 (2021-09-24)
+
+- Fixed the currency column type to use right alignment. [#147](https://github.com/blackbaud/skyux-ag-grid/pull/147)
```

Comment:
```suggestion
- Fixed the currency column type so the values are right-aligned. [#147](https://github.com/blackbaud/skyux-ag-grid/pull/147)
```
Suggested the change because "right" could also be read as proper or correct.